packages/dd-trace/src/exporters/common/request.js

```diff
@@ -15,20 +16,28 @@ const httpAgent = new http.Agent({ keepAlive, maxTotalSockets })
 const httpsAgent = new https.Agent({ keepAlive, maxTotalSockets })
 const containerId = docker.id()
 
+const isForm = (data) => {
+  return data instanceof Readable
+}
+
 let activeRequests = 0
 
 function request (data, options, keepAlive, callback) {
   if (!options.headers) {
     options.headers = {}
   }
 
+  const isFormData = isForm(data)
```

Comment:
Why not just inline the function? It checks if it's readable, not if it's FormData.